### ISSUE TITLE
MultiComboBox: A11Y | JAWS Screen reader - Repetitive announcements should be avoided

### ISSUE THREAD

**Issue by t3s3r4ct** [SAP/ui5-webcomponents#11394]
### Describe the bug

Note: This may be more appropriate as a consulting issue.

Using the screen reader JAWS 2025, there are redundant announcements when using the MultiComboBox component and items.

`Multiple selection mode` is announced for each item in the combo box list. But `Contains Multi-Selectable Items list` announcement is already made, leading to repetitive announcements.

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/inputs-multicombobox--docs

### Reproduction steps

1. Open the speech to text for JAWS
2. Go to the storybook for MultiComboBox
3. There are multiple repetitive announcements indicating the combo box list is multi-selection

### Expected Behaviour

Since "Contains Multi-Selectable Items list" announcement is made for the combo box, the "Multiple selection mode" announcement for all list items is repetitive and can be avoided.

### Screenshots or Videos

![Image](https://github.com/user-attachments/assets/fb9536fb-d86a-40fe-a08b-75a5ee6d8715)

![Image](https://github.com/user-attachments/assets/2b384ca9-eda2-4592-8911-194879a2d823)

### UI5 Web Components for React Version

2.9.0

### UI5 Web Components Version

2.9.0

### Browser

Edge

### Relevant log output

```shell

```

### Organization

SAP

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by TeodorTaushanov** [comment]
Hello @SAP/ui5-webcomponents-topic-rl, can you check this question?

**Comment by ivoplashkov** [comment]
Hello team,

The issue arises from the checkbox being focusable, which is not the expected behavior within the context of the MultiComboBox component. This same issue has already been addressed in UI5, as shown in the following commit: https://github.com/SAP/openui5/commit/2e8fec7d9dfd421672d9d5b4a11272b848b5f244.

We need to implement a similar fix to ensure the checkbox is no longer focusable in our implementation.

Best regards,
Ivaylo
